src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1213,6 +1215,57 @@ public void readonly() {
     sendCommand(READONLY);
   }
 
+  public void geoadd(byte[] key, double longitude, double latitude, byte[] member) {
+    sendCommand(GEOADD, key, toByteArray(longitude), toByteArray(latitude), member);
+  }
+
+  public void geoadd(byte[] key, Map<byte[], GeoCoordinate> memberCoordinateMap) {
```

Comment:
@marcosnils 
Yes, geo stores datas to sorted set internally. That's why I uses Map<byte[], GeoCoordinate> to represent multiple members of geoadd.
zadd() also uses Map<byte[], GeoCoordinate> cause score is changeable and only member name is unique. 
Set<GeoCoordinate> allows duplication of same member with different coordinate, and we can't guess which thing will be stored effectively.